sdc/tests/tests_perf/test_perf_series.py

```diff
@@ -97,8 +97,8 @@ def _test_case(self, pyfunc, name, total_data_length, input_data=None, data_num=
     TC(name='getitem_idx_bool_array', size=[10 ** 7], call_expr='A[B]', usecase_params='A, B',
        data_gens=(gen_series, partial(gen_arr_of_dtype, dtype='bool', random=False)),
        input_data=[None, [True, False, False, True, False, True]]),
-    TC(name='getitem_idx_int_series', size=[10 ** 7], call_expr='A[B]', usecase_params='A, B',
-       data_gens=(gen_series, partial(gen_series, dtype='int', limits=(0, 10 ** 7)))),
+   #  TC(name='getitem_idx_int_series', size=[10 ** 7], call_expr='A[B]', usecase_params='A, B',
```

Comment:
Maybe add a comment here why it's commented (just to not forget what to fix before uncommenting)?

Reply:
Good advice. Unfortunately saw this after merge.

Reply:
There still will be another rc with docs and infra fixes I think, you can fix in separate PR